worker/raft/raftclusterer/worker_test.go

```diff
@@ -268,6 +268,83 @@ func (s *WorkerSuite) TestRequestsDetails(c *gc.C) {
 	}
 }
 
+func (s *WorkerSuite) TestDisappearingAddresses(c *gc.C) {
```

Comment:
Are we missing a test for the case where the address had not yet been seen and we get a blank one?